sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 ## Release History
 
-### 4.3.1b2 (Unreleased)
+### 4.3.1 (Unreleased)
 #### Features Added
  - Added `correlated_activity_id` for query operations
  - Added cross regional retries for Service Unavailable/Request Timeouts for read/Query Plan operations
+ - GA release of CosmosHttpLoggingPolicy and autoscale feature
```

Comment:
a question for future: add we reference to the PR link ? has found that very helpful

Reply:
We had a discussion with the Central SDK team and ended up deciding not to since the CHANGELOG and READMEs are both meant to be for customers as well - I do agree that it would be useful though, perhaps we will need to keep track of it in a separate document within the SDK